tests/tracing/test_apollotracing.py

```diff
@@ -0,0 +1,70 @@
+import pytest
+from graphql import get_introspection_query
+
+from ariadne import graphql
+from ariadne.contrib.tracing.apollotracing import ApolloTracingExtension
+
+
+@pytest.mark.asyncio
+async def test_apollotracing_extension_causes_no_errors_in_query_execution(schema):
+    _, result = await graphql(
+        schema, {"query": "{ status }"}, extensions=[ApolloTracingExtension]
+    )
+    assert result["data"] == {"status": True}
+
+
+@pytest.mark.asyncio
+async def test_apollotracing_extension_adds_tracing_data_to_result_extensions(schema):
+    _, result = await graphql(
+        schema, {"query": "{ status }"}, extensions=[ApolloTracingExtension]
+    )
+    assert result["extensions"]["tracing"]["version"] == 1
+    assert result["extensions"]["tracing"]["startTime"]
+    assert result["extensions"]["tracing"]["endTime"]
+    assert result["extensions"]["tracing"]["duration"]
+    assert result["extensions"]["tracing"]["execution"]
+
+
+@pytest.mark.asyncio
+async def test_apollotracing_extension_adds_resolvers_timing_in_result_extensions(
+    schema
+):
+    _, result = await graphql(
+        schema, {"query": "{ status }"}, extensions=[ApolloTracingExtension]
+    )
+
+    resolvers = result["extensions"]["tracing"]["execution"]["resolvers"]
+
+    assert len(resolvers) == 1
+    assert resolvers[0]["path"] == ["status"]
```

Comment:
Maybe we can use snapshots in those tests?
If there is problem with time and duration results we could mock _time_ fetching or use freezegun.